ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -229,6 +235,7 @@ function ViolationsTablePage(): ReactElement {
                 const parsedMessage = getAxiosErrorMessage(error);
                 setCurrentPageAlertsErrorMessage(parsedMessage);
                 setIsLoadingAlerts(false);
+                setIsTableDataUpdating(false);
```

Comment:
We can probably add a `setIsTableDataUpdating(false)` in a `.finally()` to execute regardless of the result.

Reply:
Good idea, updated